tests/ddos/dddo_without_service.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+json_dict = {
```

Comment:
creating this is superfluous. First, there is a typo in the file, saying dddo with 3 "d"s instead of 2. Second, please reuse the normal ddo, copy it using deepcopy and just pop the service. Isn't it easier than adding another file we need to maintain?

Reply:
@mariacarmina, can you please address this?